.travis.yml

```diff
@@ -49,4 +31,4 @@ deploy:
     target-branch: gh-pages
     on:
       branch: master
-      python: 3.7
+      python: 3.6
```

Comment:
why this change?

Reply:
Because there is no advantage on one over the other, but this one is proved to work with the upper configuration, while 3.7 hasn't.

Reply:
I realized I ended up upgrading to 3.7 in the other project where I had used this configuration, so I reverted it to 3.7 also here.